src/fklearn/training/classification.py

```diff
@@ -446,7 +446,7 @@ def nlp_logistic_classification_learner(df: pd.DataFrame,
     """
 
     # set default params
-    default_vect_params = {"strip_accents": "unicode", "min_df": 20}
+    default_vect_params = {"strip_accents": "unicode", "min_df": 0}
```

Comment:
[scikit-learn default]( https://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.TfidfVectorizer.html) is actually min_df=1.
```suggestion
    default_vect_params = {"strip_accents": "unicode", "min_df": 1}
```

Reply:
you are right, it's max_df=1, thank you!